### ISSUE TITLE
Suggestions and Changes That Should be Done To Make this Index More Dynamic

### ISSUE THREAD

**Issue by AmirulAndalib** [cheems/goindex-extended#21]
- [ ]  **`1`**  **move from Dplayer To Playr -- cuz Dplayer base repo Stuck on 2020 while Plyr is updated once/twice in a Week**

- [ ]  **`2`**  **add icons for video audio document pdf docs etc etc like Bhadoos index**

- [ ]  **`3`**  **Add service account support**

- [ ]  **`4`**  **fix themes**

- [ ]  **`5`** **If Possible Add support for `TMDB` and `Anilist` api support**

- [ ]  **`6`**  **Add the ability to set whether google drive videos will be transcoded or not (by deault it transcoded i guess)**

## You can check [Bhadoos repo](https://gitlab.com/ParveenBhadooOfficial/Google-Drive-Index) and [Libdrive Repo](https://github.com/libDrive) for the suggestions i said above

# thank you

**Comment by cheems** [comment]
First three suggestions are already in the todo list. Since I'm not familiar with any movie database api like TMDB, I cannot say it will be implemented soon.

Even though I'm interested with the 6th suggestion, it seems like the official API doesn't support that feature. But I found that calling `docs.google.com/get_video_info?docid=FILE_ID` returns a file which includes links to the videos in different resolutions. And probably it may be the same method that Libdrive uses. Since It doesn’t seem to be actually documented anywhere, I have to go through the source code of the repos which have mentioned the feature including the repo you have mentioned. So, it will take some time.

And also I would like to know which kind of things should be fixed when it comes to themes.

**Comment by AmirulAndalib** [comment]
> First three suggestions are already in the todo list. Since I'm not familiar with any movie database api like TMDB, I cannot say it will be implemented soon.
> 
> Even though I'm interested with the 6th suggestion, it seems like the official API doesn't support that feature. But I found that calling `docs.google.com/get_video_info?docid=FILE_ID` returns a file which includes links to the videos in different resolutions. And probably it may be the same method that Libdrive uses. Since It doesn’t seem to be actually documented anywhere, I have to go through the source code of the repos which have mentioned the feature including the repo you have mentioned. So, it will take some time.
> 
> And also I would like to know which kind of things should be fixed when it comes to themes.

@cheems  yes You are correct resolution selection removes the transcoding But The Main  definition of transcoding Is To Make almost all  video Playable in website But with lower resolution and Bad Codecs...But i think Transcoding is not needed as People Can Use External players to Play the Unaplayable Video But You shall Remove Transcoding and Add Resoultion Selection If possible....


Also For themes I think you Should Not follow Bhadoos Repo Instead I want Themes Like Tks-gindex-v4 themes etc... Which has Netflix Like Theme and also gives a Netflix feeling....

and for tmdb i think You shall Follow Tksgindex-v4 it might be easier for you ...Idk for anilist But anilist is on libdrive/server repo....

**Comment by cheems** [comment]
I will have a look. After completing the goals related to the backend, will head to experiment things related to the front end.
